src/filesystem/impls/appshell/node/FileWatcherDomain.js

```diff
@@ -99,7 +99,8 @@ function watchPath(path, ignored) {
             ignoreInitial: true,
             ignorePermissionErrors: true,
             followSymlinks: true,
-            ignored: ignored
+            ignored: ignored,
+            usePolling: process.platform === "win32"
```

Comment:
There's no `fsevents` on Windows so polling is required, right? That is unless we continue to use `fsevents-win` 🐱